src/main/groovy/org/checkerframework/gradle/plugin/CheckerFrameworkPlugin.groovy

```diff
@@ -116,12 +118,17 @@ final class CheckerFrameworkPlugin implements Plugin<Project> {
       jdkVersion = ANNOTATED_JDK_NAME_JDK8
     }
 
+    // Check whether to use the Error Prone javac
+    def needErrorProneJavac =
+      javaVersion.java8 && (${LIBRARY_VERSION}.tokenize(".")[0].toInteger() >= 3)
```

Comment:
@kelloggm Is `LIBRARY_VERSION` the right variable to use to determine the current CF version being used?
This would avoid unzipping the jar file, which you had mentioned is causing problems.

Reply:
`LIBRARY_VERSION` is the default version that is used if the user did not specify a Checker Framework version. Checking it here is insufficient, because the behavior will be the same regardless of what version was chosen by the user. I'll try to build off of this to actually check the right value.